setup.py

```diff
@@ -139,4 +140,7 @@ def get_extensions():
     long_description_content_type="text/markdown",
     url="https://github.com/pytorch-labs/ao",
```

Comment:
can you remove `-labs` as well

Reply:
Will do as a separate PR, as that isn't related to this change